### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by ng29**
Translated the Tutorial/part-five successfully. Please check and review. @spielers / @visshaljagtap.
Issue #1

### CODE REVIEW

File: docs/tutorial/part-five/index.md [hunk 1]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-> This tutorial is part of a series about Gatsby’s data layer. Make sure you’ve gone through [part 4](/tutorial/part-four/) before continuing here.
+> हे ट्यूटोरियल गॅट्सबीच्या डेटा लेयर बद्दलच्या मालिकेचा एक भाग आहे. आपण जात असल्याचे सुनिश्चित करा [part 4](/tutorial/part-four/) येथे सुरू ठेवण्यापूर्वी.
```
Comment (spielers) [inline docs/tutorial/part-five/index.md]: गॅट्सबीच्या should be in English as it is !

**Review by spielers — CHANGES_REQUESTED** [review]
Hey, @ng29  Thanks for your contribution but please retranslate the entire document keeping Marathi grammar in mind Just copy-pasting from google translate won't work here

And also let me know if you are aiming for hacktoberfest 

Best,
spielers

**Comment by ng29** [comment]
> Hey, @ng29 Thanks for your contribution but please retranslate the entire document keeping Marathi grammar in mind Just copy-pasting from google translate won't work here
> 
> And also let me know if you are aiming for hacktoberfest
> 
> Best,
> spielers

ooh, i thought that translation will be suffice :(
And yes was contributing towards Hacktoberfest...

File: docs/tutorial/part-five/index.md [hunk 2]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
 
-## What's in this tutorial?
+## या ट्यूटोरियल मध्ये काय आहे?
 
-In this tutorial, you'll be learning about how to pull data into your Gatsby site using GraphQL and source plugins. Before you learn about these plugins, however, you'll want to know how to use something called GraphiQL, a tool that helps you structure your queries correctly.
+या ट्यूटोरियल मध्ये, आपण ग्राफिक्स आणि स्त्रोत प्लगइन वापरुन आपल्या गॅटस्बी साइटमध्ये डेटा कसा काढायचा याबद्दल शिकत आहात. तथापि आपण या प्लगइन्सबद्दल जाणून घेण्यापूर्वी, आपल्याला ग्राफिक नावाचे काहीतरी कसे वापरावे हे जाणून घ्यायचे आहे, जे आपले क्वेरी अचूकपणे रचण्यात मदत करते.
```
Comment (visshaljagtap) [inline docs/tutorial/part-five/index.md]: ग्राफिक -> GraphiQL

 a tool that helps you structure your queries correctly. -> एक साधन जे आपणास आपल्या क्वेरीची योग्यरित्या रचना करण्यात मदत करते.

File: docs/tutorial/part-five/index.md [hunk 3]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
 
-## Introducing GraphiQL
+## सादर करीत आहोत ग्राफीक्यूएल
```
Comment (visshaljagtap) [inline docs/tutorial/part-five/index.md]: dont translate GraphiQL

**Review by visshaljagtap — CHANGES_REQUESTED** [review]

File: docs/tutorial/part-five/index.md (hunk 3 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: done

File: docs/tutorial/part-five/index.md (hunk 1 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 2 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md [hunk 4]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
+## या ट्यूटोरियल मध्ये काय आहे?
 
-In this tutorial, you'll be learning about how to pull data into your Gatsby site using GraphQL and source plugins. Before you learn about these plugins, however, you'll want to know how to use something called GraphiQL, a tool that helps you structure your queries correctly.
+या ट्यूटोरियल मध्ये, आपण ग्राफिक्स आणि स्त्रोत प्लगइन वापरुन आपल्या गॅटस्बी साइटमध्ये डेटा कसा काढायचा याबद्दल शिकत आहात. तथापि आपण या प्लगइन्सबद्दल जाणून घेण्यापूर्वी, आपल्याला GraphiQL नावाचे काहीतरी कसे वापरावे हे जाणून घ्यायचे आहे, एक साधन जे आपणास आपल्या क्वेरीची योग्यरित्या रचना करण्यात मदत करते.
 
-## Introducing GraphiQL
+## सादर करीत आहोत GraphiQL
 
-GraphiQL is the GraphQL integrated development environment (IDE). It's a powerful (and all-around awesome) tool you'll use often while building Gatsby websites.
+ग्राफीक्यूएल म्हणजे ग्राफिक एकात्मिक विकास वातावरण (आयडीई). हे एक शक्तिशाली (आणि सर्वत्र अद्भुत) साधन आहे जे आपण गॅटस्बी वेबसाइट्स तयार करताना वारंवार वापरता.
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: ग्राफीक्यूएल -> GraphiQL
GraphQL integrated development environment -> GraphQl इंटिग्रेटेड डेव्हलपमेंट एंवीरोन्मेन्ट

File: docs/tutorial/part-five/index.md [hunk 5]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
+
+जेव्हा आपल्या साइटचा विकास सर्व्हर चालू असतो तेव्हा आपण त्यात प्रवेश करू शकता - सामान्यपणे
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: विकास -> डेव्हलपमेंट

File: docs/tutorial/part-five/index.md [hunk 6]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
 `http://localhost:8000/___graphql`.
 
 <video controls="controls" autoplay="true" loop="true">
   <source type="video/mp4" src="/graphiql-explore.mp4" />
-  <p>Your browser does not support the video element.</p>
+  <p>आपला ब्राउझर व्हिडिओ घटकास समर्थन देत नाही.</p>
 </video>
 
-Poke around the built-in `Site` "type" and see what fields are available on it -- including the `siteMetadata` object you queried earlier. Try opening GraphiQL and play with your data! Press <kbd>Ctrl + Space</kbd> (or use <kbd>Shift + Space</kbd> as an alternate keyboard shortcut) to bring up the autocomplete window and <kbd>Ctrl + Enter</kbd> to run the GraphQL query. You'll be using GraphiQL a lot more through the remainder of the tutorial.
+अंगभूत सुमारे झोके `Site` "प्रकार" आणि त्यावर कोणती फील्ड उपलब्ध आहेत ते पहा `siteMetadata` ऑब्जेक्ट आपण आधी चौकशी केली. ग्राफीक्यूएल आणि उघडण्याचा प्रयत्न करा आपल्या डेटासह खेळा! दाबा <kbd>Ctrl + Space</kbd> (किंवा वापरा <kbd>Shift + Space</kbd> वैकल्पिक कीबोर्ड शॉर्टकट म्हणून) स्वयंपूर्ण विंडो आणण्यासाठी आणि <kbd>Ctrl + Enter</kbd> ग्राफिक क्वेरी चालवण्यासाठी. ट्युटोरियलच्या उर्वरित भागातून आपण आणखी बरेच काही ग्राफिक्यूएल वापरणार आहात.
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: please dont translate GraphQL and GraphiQL

File: docs/tutorial/part-five/index.md [hunk 7]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
+गॅटस्बी साइट मधील डेटा कोठूनही येऊ शकतोः एपीआय, डेटाबेस, सीएमएस, स्थानिक फायली इ.
 
-Source plugins fetch data from their source. E.g. the filesystem source plugin knows how to fetch data from the file system. The WordPress plugin knows how to fetch data from the WordPress API.
+स्त्रोत प्लगइन त्यांच्या स्त्रोतांमधून डेटा आणतात. उदा. फाइल सिस्टम स्रोत प्लगइनला फाइल सिस्टममधून डेटा कसा आणला जावा हे माहित असते. वर्डप्रेस प्लगइनला वर्डप्रेस एपीआय वरून डेटा कसा आणता येईल हे माहित आहे.
 
-Add [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) and explore how it works.
+जोडा [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) आणि ते कसे कार्य करते ते एक्सप्लोर करा.
 
-First, install the plugin at the root of the project:
+प्रथम, प्रकल्पाच्या मुळाशी प्लगइन स्थापित करा.
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: स्थापित -> इन्स्टॉल

File: docs/tutorial/part-five/index.md [hunk 8]
```diff
@@ -4,50 +4,51 @@ typora-copy-images-to: ./
...
 
 ```shell
 npm install --save gatsby-source-filesystem
 ```
 
-Then add it to your `gatsby-config.js`:
+मग ते आपल्यास जोडा `gatsby-config.js`:
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: मग ते आपल्या `gatsby-config.js` मध्ये ऍड करा :

File: docs/tutorial/part-five/index.md [hunk 9]
```diff
@@ -75,42 +76,42 @@ module.exports = {
 }
 ```
 
-Save that and restart the gatsby development server. Then open up GraphiQL again.
+ते सेव्ह करा आणि गॅट्सबी डेव्हलपमेंट सर्व्हर रीस्टार्ट करा. मग पुन्हा ग्रॅफिक्यूएल उघडा.
 
-In the explorer pane, you'll see `allFile` and `file` available as selections:
+एक्सप्लोरर उपखंडात, आपण पहाल `allFile` आणि `file` निवडी म्हणून उपलब्ध:
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: उपखंडात -> पेन

**Review by vishal-vc — CHANGES_REQUESTED** [review]

File: docs/tutorial/part-five/index.md (hunk 4 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 5 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 9 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 8 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 7 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified

File: docs/tutorial/part-five/index.md (hunk 6 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: modified, thanks for sugggesting

**Comment by ng29** [comment]
Hi @visshaljagtap ,  

All requested changes have been catered, kindly review and merge (If no more changes required).

Happy Hacktoberfest

File: docs/tutorial/part-five/index.md [hunk 10]
```diff
@@ -75,42 +76,42 @@ module.exports = {
...
+क्वेरी पुन्हा चालविण्यासाठी प्रत्येक वेळी. आपण अद्यतनित क्वेरी परिणाम दिसेल:
 
 ![allfile-query](allfile-query.png)
 
-The result is an array of `File` "nodes" (node is a fancy name for an object in a
-"graph"). Each `File` node object has the fields you queried for.
+याचा परिणाम अ‍ॅरे आहे `File` "नोड्स" (नोड ए मधील ऑब्जेक्टसाठी एक काल्पनिक नाव आहे
+"आलेख"). प्रत्येक `File` नोड ऑब्जेक्टमध्ये आपण फिल्डसाठी क्वेरी केली आहे.
+
+## ग्राफिक क्वेरीसह एक पृष्ठ तयार करा
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: please replace पृष्ठ with पेज.

it is everywhere applicable

File: docs/tutorial/part-five/index.md [hunk 11]
```diff
@@ -142,18 +143,17 @@ export const query = graphql`
...
+जेणेकरून आपण यूआय तयार करताना आपल्या ब्राउझर कन्सोलमधील डेटा एक्सप्लोर करू शकता.
 
-If you visit the new page at `/my-files/` and open up your browser console
-you will see something like:
+आपण नवीन पृष्ठास भेट दिली तर `/my-files/` आणि आपला ब्राउझर कन्सोल उघडा आपल्याला असे काहीतरी दिसेल:
 
 ![data-in-console](data-in-console.png)
 
-The shape of the data matches the shape of the GraphQL query.
+डेटाचा आकार ग्राफिक क्वेरीच्या आकाराशी जुळतो.
```
Comment (vishal-vc) [inline docs/tutorial/part-five/index.md]: please dont translate GraphQL, it is applicable every where, please update

**Review by vishal-vc — CHANGES_REQUESTED** [review]

File: docs/tutorial/part-five/index.md (hunk 11 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: SURE

File: docs/tutorial/part-five/index.md (hunk 10 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: Modifying

Comment (ng29) [reply docs/tutorial/part-five/index.md]: done

File: docs/tutorial/part-five/index.md (hunk 11 above)
Comment (ng29) [reply docs/tutorial/part-five/index.md]: all done

**Comment by spielers** [comment]
Hey, @ng29  Thanks for your contribution!

Best,
spielers

**Comment by ng29** [comment]
> Hey, @ng29 Thanks for your contribution!
> 
> Best,
> spielers

Glad to contribute @spielers , just one query that am i eligible for Gatsby Hacktoberfest 2020? ?

Cheers

**Comment by ng29** [comment]
hi @spielers  / @vishal-vc  / @visshaljagtap 
When my PR will be merged?

**Comment by spielers** [comment]
@visshaljagtap Good to go?

**Comment by visshaljagtap** [comment]
LGTM!

**Review by visshaljagtap — APPROVED** [review]

**Comment by ng29** [comment]
> hi @spielers / @vishal-vc / @visshaljagtap
> When my PR will be merged?

Hey @spielers .

I reckon you miss merging this request :(

**Comment by ng29** [comment]
Hi @vishal-vc  / @spielers  / @visshaljagtap 
Please merge this PR, as it is pending since long

**Review by spielers — APPROVED** [review]

**Comment by spielers** [comment]
Hey, @ng29 Thanks for your valuable contributions!

**Pull request merged**